gazel.go

```diff
@@ -506,6 +563,34 @@ func ReconcileRules(path string, rules []*bzl.Rule, dryRun bool) (bool, error) {
 	return writeFile(path, f, true, dryRun)
 }
 
+func reconcileLoad(f *bzl.File, rules []*bzl.Rule) {
+	usedRuleKindsMap := map[string]bool{}
+	for _, r := range rules {
+		usedRuleKindsMap[r.Kind()] = true
+	}
+
+	usedRuleKindsList := []string{}
+	for k, _ := range usedRuleKindsMap {
+		usedRuleKindsList = append(usedRuleKindsList, k)
+	}
+	sort.Strings(usedRuleKindsList)
+
+	for _, r := range f.Rules("load") {
+		const goRulesLabel = "@io_bazel_rules_go//go:def.bzl"
+		args := bzl.Strings(&bzl.ListExpr{List: r.Call.List})
+		if len(args) == 0 {
+			continue
+		}
+		if args[0] != goRulesLabel {
+			continue
+		}
+		r.Call.List = asExpr(append(
```

Comment:
You can load("@io_bazel_rules_go//go:def.bzl") later on in the file and only the first load() is managed. a bit hacky i know, but open to alternatives.